### PR TITLE
Adds Linux -- Terminology and GNOME support

### LINKED ISSUE #7: Linux Port
What would be needed to re-do/change in order for a Linux port? How much is the current 'method' tied to iTerm2/macOSx? Is this an iTerm2 feature?

Because AFAIK, images on Linux terminals are a deadly sin, except maybe for terminology... So, what kind of work needs doing till I can add this to my rice :smiley:

### PR DESCRIPTION
**Pull request by samosaara**
Works here. https://streamable.com/41jg2 and closes #7 I disabled transparency to take the prints but it works. And I actually needed some testing from a MacOS host to see if I didn't end up breaking anything.

### CODE REVIEW

File: scripter.py [hunk 1]
```diff
@@ -62,31 +64,52 @@ def __create_wallpaper_bash():
...
+        # Create, save and run the bash script to change the terminal background.
+        __iterm2_create_terminal_script(pokemon)
+        __darwin_create_terminal_bash()
+        os.system(cwd + "/./Scripts/run.sh")
+    if sys.platform == "linux":
+        os.system(__linux_create_terminal(pokemon))
+
+def __linux_create_terminal(pokemon):
+    if os.environ.get("TERMINOLOGY") == '1':
+        return "tybg \"" + pokemon.get_path() + "\""
```
Comment (samosaara) [inline scripter.py]: Lots of other if's to other terminals go here.

File: scripter.py [hunk 2]
```diff
@@ -62,31 +64,52 @@ def __create_wallpaper_bash():
...
+        os.system(cwd + "/./Scripts/run.sh")
+    if sys.platform == "linux":
+        os.system(__linux_create_wallpapper_script(pokemon))
+
+def __linux_create_wallpapper_script(pokemon):
+    # If its gnome... aka GDMSESSION=gnome-xorg, etc.
+    if os.environ.get("GDMSESSION").find("gnome") >= 0:
+        return "gsettings set org.gnome.desktop.background picture-uri " + \
+            "\"file://"+ pokemon.get_path()+"\""
+    #elif condition of KDE...
```
Comment (samosaara) [inline scripter.py]: And lots of ifs to other WM's go here.

File: scripter.py [hunk 3]
```diff
@@ -62,31 +64,52 @@ def __create_wallpaper_bash():
...
     __determine_pokemon(db, "background.scpt")
 
 
+# Print the current Pokemon that is being used the wallpaper.
 def determine_wallpaper_pokemon(db):
-    # Print the current Pokemon that is being used the wallpaper.
     __determine_pokemon(db, "wallpaper.scpt")
 
 
+# Helper method to get the current Pokemon that is in the specified script.
```
Comment (samosaara) [inline scripter.py]: Autism. :+1:

**Comment by charlesmilette** [comment]
That's nice, but what about the default `gnome-terminal`

**Comment by samosaara** [comment]
That's a feature exclusive to the **ubuntu** version of the gnome terminal witch I don't use nor have access to (it's heavily tied to unity 7 witch will be ditched soon along with it's terminal...) I'm on arch linux. Sorry. But feel free to build upon my code, technically should be an elif away.

**Comment by charlesmilette** [comment]
Don't feel sorry for being on Arch, I too am using it :P 

I've been considering changing terminals away from it and terminator seems like a good choice.

**Comment by LazoCoder** [comment]
I'll test this on my mac tonight and accept the pull request if its all good. Thank you for your contribution!

**Pull request merged**